components/match_ticker/commons/match_ticker.lua

```diff
@@ -359,13 +447,20 @@ function MatchTicker:adjustMatch(match)
 	return match
 end
 
+--- Will only switch if there are exactly 2 opponents
+---@param match table
+---@return table
 function MatchTicker.switchOpponents(match)
+	if #match.opponents ~= 2 then
+		return match
+	end
 	local winner = tonumber(match.winner) or 0
 	match.winner = winner == 1 and 2
 		or winner == 2 and 1
 		or match.winner
 
 	match.match2opponents[1], match.match2opponents[2] = match.match2opponents[2], match.match2opponents[1]
```

Comment:
this line still needed?

Reply:
If the function is called with more than 2 opponents, weird shit starts happening in the data, felt good to add it there

Reply:
don't you check for only 2 oponents at the top of the function

i am meaning this line
`match.match2opponents[1], match.match2opponents[2] = match.match2opponents[2], match.match2opponents[1]`
i think it is redundant since you switched everything to `.opponents` instead of `.match2opponents`

Reply:
Probably not. But we still have that data, so should be swapped around. In case future dev works uses that.